build.sbt

```diff
@@ -11,7 +11,7 @@ inThisBuild(Seq(
   ),
   scmInfo := Some(ScmInfo(url("https://github.com/lightbend/mima"), "scm:git:git@github.com:lightbend/mima.git")),
   dynverVTagPrefix := false,
-  scalacOptions := Seq("-feature", "-deprecation", "-Xlint"),
+  scalacOptions ++= Seq("-feature", "-Xsource:3", "-Xlint", "-Wconf:cat=deprecation&msg=Stream|JavaConverters:s"),
```

Comment:
Does that deprecation warning keep the rest of the deprecations showing?  Intuitively I would have thought there should be an "enable deprecations" option in addition to the "oh and silence these ones", which has me confused.

Reply:
-Xlint enables deprecation, and that was backported to 2.12 as well (as I did check).